examples/java/src/test/java/dev/selenium/elements/InformationTest.java

```diff
@@ -1,7 +1,71 @@
 package dev.selenium.elements;
 
-import dev.selenium.BaseTest;
+import org.junit.jupiter.api.Test;
+import org.openqa.selenium.By;
+import org.openqa.selenium.Rectangle;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import java.time.Duration;
+import static org.junit.jupiter.api.Assertions.assertEquals;
 
-public class InformationTest extends BaseTest {
+public class ElementInformation {
```

Comment:
Shouldn't the class name match the file name?

Why don't you use the `BaseTest` class?

Reply:
@diemol made the change. sorry for this. that day was a bad day, when i originally created this. apologies for the inconvenience.

Reply:
Thank you. No need to apologize at all!